### PR TITLE
edencommon fb303 fbthrift fizz folly wangle watchman 2023.06.12.00

### PR DESCRIPTION
**Pull request by razvanazamfirei**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Edencommon + friends

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Undefined symbols for architecture arm64:
    "_softaes_block_encrypt", referenced from:
        _aegis128l_update in aead_aegis128l_soft.c.o
        _aegis256_encrypt_detached in aead_aegis256_soft.c.o
        _aegis256_decrypt_detached in aead_aegis256_soft.c.o
        _aegis256_update in aead_aegis256_soft.c.o
  ld: symbol(s) not found for architecture arm64
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by razvanazamfirei** [comment]
fizz failing due to `libsodium AEGIS` which was added in the last release.

File: Formula/fizz.rb [hunk 1]
```diff
@@ -31,13 +31,16 @@ class Fizz < Formula
   depends_on "zstd"
 
   fails_with gcc: "5"
-
   def install
+    inreplace "fizz/CMakeLists.txt",
+    "set(FIZZ_SOURCES\n",
+    "set(FIZZ_SOURCES\n third-party/libsodium-aegis/private/softaes.c"
+
```
Comment (razvanazamfirei) [inline Formula/fizz.rb:38]: There must be a saner way of doing this.

Comment (carlocab) [reply Formula/fizz.rb:38]: Weird. Is this code that depends on an unreleased version of `libsodium`, or something?

Comment (carlocab) [reply Formula/fizz.rb:38]: I suggest upstreaming a patch. Seems like they just forgot to add this here: https://github.com/facebookincubator/fizz/commit/ecab8fc4434667387db4556afbe05062636be58e

Comment (razvanazamfirei) [reply Formula/fizz.rb:38]: Looks like someone else got there first: https://github.com/facebookincubator/fizz/commit/0dc415e2e7dade586b445946a939d4f8ff15e8d2

Will wait for the next release

Comment (carlocab) [reply Formula/fizz.rb:38]: Cool, we can backport that then. Will push an update.

**Review by carlocab — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request merged**